olive/cli/auto_opt.py

```diff
@@ -267,19 +304,49 @@ def _get_passes_config(self, config: Dict[str, Any], olive_config: OlivePackageC
             if value is not None:
                 set_nested_dict_value(passes_config, keys, value)
 
-        del passes_config["conversion" if self.args.use_model_builder else "model_builder"]
-        if (self.args.provider != "QNNExecutionProvider") or self.args.use_model_builder:
+        passes_to_remove = set()
+        if config["input_model"]["type"].lower() == "onnxmodel":
+            # remove passes that only operate on PyTorch/Hf models
+            passes_to_remove.update(["capture_split_info", "conversion", "model_builder", "split_model"])
+        else:
+            # only one graph capture pass is used
+            passes_to_remove.add("conversion" if self.args.use_model_builder else "model_builder")
```

Comment:
We want to provide the optimum exporter as an option for issues like Neg_*

Reply:
should I add another option like `use_optimum_conversion`? Or use it by default for conversion with hf-models?

Reply:
Another option --use_optimum_conversion

Reply:
https://onnxruntime.ai/docs/tutorials/web/ep-webgpu.html

Reply:
not adding this option for now as discussed offline. More work is needed if we want to use optimum converter for models with adapters.